vertical-pod-autoscaler/pkg/recommender/model/cluster.go

```diff
@@ -369,6 +382,22 @@ func (cluster *ClusterState) RecordRecommendation(vpa *Vpa, now time.Time) error
 	return nil
 }
 
+// GetActiveMatchingPods returns a list of currently active pods that match the
```

Comment:
This is a potentially heavy function. I can see it being used now only for logging an erroneous situation but I'd like to make sure it won't become accidentally used elsewhere.
Maybe a comment that it should be used sparingly would be good?

Reply:
Done.